composer.json

```diff
@@ -18,7 +18,7 @@
         "phpstan-81": "phpstan/vendor/bin/phpstan -cphpstan/php81.neon analyse",
         "phpstan-82": "phpstan/vendor/bin/phpstan -cphpstan/php82.neon analyse",
         "phpstan-83": "phpstan/vendor/bin/phpstan -cphpstan/php83.neon analyse",
-        "tests-unit": "./vendor/bin/phpunit tests",
+        "phpunit": "./vendor/bin/phpunit tests",
```

Comment:
proč ta změna na phpunit? Nejsem moc zastánce, když se věci jmenují podle technologie. A když už tak bych to dal aspoň phpunit-unit.. protože pak může být phpunit-integration, phpunit-performance, phpunit-coverage...

A navíc koukám, že tests-coverage podtím zůstal a teď je z toho kočkopes :smiley:

Reply:
It's about consistency in all packeta projects. 

I just temporarily reverted.